boards/phytec/phyboard_nash/doc/index.rst

```diff
@@ -0,0 +1,178 @@
+.. _phyboard_nash:
+
+PhyBOARD Nash (NXP i.MX93)
+##########################
+
+Overview
+********
+
+The phyBOARD Nash is based on the phyCORE-i.MX93 SoM is based on the NXP i.MX93
+SoC. It features common industrial interfaces and can be used as a reference for
+development or in the final product. It is an entry-level development board,
+which helps developers to get familiar with the module before investing a large
+amount of resources in more specific designs.
+
+i.MX93 MPU is composed of one cluster of 2x Cortex-A55 cores and a single
+Cortex®-M33 core. Zephyr OS is ported to run on one of the Cortex®-A55 core as
```

Comment:
I would remove these trademark symbols here. You're not consistent with them and I don't see any requirement here.

Reply:
Yes, removed it.